modyn/trainer_server/internal/dataset/online_dataset.py

```diff
@@ -53,23 +47,18 @@ def __init__(
         self._transform_list: list[Callable] = []
         self._transform: Optional[Callable] = None
         self._storagestub: StorageStub = None
-        self._selectorstub: SelectorStub = None
         self._bytes_parser_function: Optional[Callable] = None
         self._num_partitions = 0
+        self._number_of_workers = number_of_workers
```

Comment:
is that still used anywhere? if not we can remove it, i didnt find any use within the OnlineDataset

Reply:
Nope, removed